### PR TITLE
[MM-34129] - Cloud 'Congratulations!' email format broken

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Gets rid of flex box in the cloud_welcome_email template

Screenshot 2021-03-22 at 22 31 27

#### Ticket Link
[MM-34129](https://mattermost.atlassian.net/browse/MM-34129?atlOrigin=eyJpIjoiMjhlYWMxMjlhMTE5NDdiMTgzM2IyMDAxNjdjNjI2YzYiLCJwIjoiaiJ9)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by marianunez — APPROVED** [review]

**Comment by AGMETEOR** [comment]
Hey @stevemudie , to be tested post merge

**Review by stevemudie — APPROVED** [review]
Will test post merge

File: templates/cloud_welcome_email.html [hunk 1]
```diff
@@ -1,116 +1,169 @@
...
-        </div>
+                                </td>
+                            </tr>
+                        </table>
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                            <tr>
+                                <td>
+                                    <p
+                                        style="font-family: Open Sans; font-style: normal; font-weight: 400; font-size: 12px; line-height: 16px; text-align: center; color: rgba(61, 60, 64, 0.56);">
+                                        © 2020 Mattermost, Inc. 855 El Camino Real, 13A-168, Palo Alto, CA, 94301
```
Comment (nickmisasi) [inline templates/cloud_welcome_email.html:155]: I think there's a translation for this (if not, let's add one). I know it's probably language agnostic but if we need to change it in the future we only have to change once

**Review by nickmisasi — APPROVED** [review]
One comment, otherwise LGTM!

**Comment by marianunez** [comment]
Merging to be able to cherry pick for cloud release on time. @AGMETEOR please address Nicks comment in a separate PR.

**Pull request merged**